lib/Storage/BeeSwarm.php

```diff
@@ -425,16 +445,6 @@ public function getMetaData($path)
 		return $data;
 	}
 
-	public function getDirectoryContent($path): \Traversable
```

Comment:
These changes were for #65 and #68. Is there any reason to revert them?

Reply:
I think we should mantain @Thesola10 version on this one

Reply:
Agreed, the original state remain.